app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/LinkPreviewViewHolder.java

```diff
@@ -77,6 +78,7 @@
     private Realm realm;
     private String url;
     private ChatMessage model;
+    private ResponseListener responseListener;
```

Comment:
No need of storing listener now, or if there is not null value, then don't reinitialize it again